core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -285,12 +288,17 @@ private List<List<KafkaConnectorModelResponse>> getPagedList(
   }
 
   private void updateTeamNamesForDisplay(List<KafkaConnectorModelResponse> topicListUpdated) {
-    topicListUpdated.forEach(
-        topicInfo -> {
-          if (topicInfo.getTeamName().length() > 9) {
-            topicInfo.setTeamName(topicInfo.getTeamName().substring(0, 8) + "...");
-          }
-        });
+    topicListUpdated.stream()
```

Comment:
@snuyanzin there is a similar method in TopicControllerService, can it be reused ?

Reply:
method in `TopicControllerService` has same issue and has signature for different types, that's why it can not be reused.

I created a more generic method which could be reused for both cases and probably for other new cases as well